checker/jdk/index/src/java/util/HashMap.java

```diff
@@ -221,9 +221,10 @@ public HashMap(@NonNegative int initialCapacity) {
      * Constructs an empty <tt>HashMap</tt> with the default initial capacity
      * (16) and the default load factor (0.75).
      */
+    @SuppressWarnings("cast.unsafe") // #1: cast from "@DoubleVal(12.0) float" to "@IntVal(12) int" will not produce any runtime error
```

Comment:
Can you please open an issue for this, and refer to the issue here in the comment?
In general, when you find a weakness, it's best to open an issue for it.

Reply:
You did not make the change I suggested.  Could you please do that?

Reply:
@mernst  I have done that now.

Reply:
Please note how I have made the comment more concise, and eliminated the inline comments that can lead to merge conflicts.